modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -535,6 +547,10 @@ Status description:: error: data exception - parsing JSON failure. Underlying er
 
 // Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
 
+=== 22NB1
+
+Status description:: error: data exception - type mismatch. Type mismatch: expected to be `{ $valueTypeOredList }` but was `{ $input }`.
```

Comment:
```suggestion
Status description:: error: data exception - type mismatch. Type mismatch: expected to be `{ $valueTypeList }` but was `{ $input }`.
```

We opted for not showing order in the docs but just document it with same list variables as in the code